media/renderer.js

```diff
@@ -10,7 +10,7 @@ const commandRouter = new CommandRouter(document, fileHandler);
 const Connection = require('./model/coreconnection');
 // eslint-disable-next-line no-unused-vars
 const coreConnection = new Connection(
-    `ws://${process.env.CORE || 'localhost:3000'}/`,
+    `ws://${process.env.SCREENCRASH_CORE || 'localhost:3000'}/`,
```

Comment:
Should we change the default to localhost:8001? I imagine we use the real core more than the dummy core now.